docs/en/table_design/expression_partitioning.md

```diff
@@ -132,6 +132,10 @@ partition_columns ::=
 
 ### Parameters
 
+:::note
+As of 3.1, only support partition by date_trunc() for date or datetime.
+:::
+
```

Comment:
do you mean that as of 3.1, expression partitioning only supports one expression, that is date_trunc() and does not support time_slice?